.github/actions/aurora-manage-cluster/README.md

```diff
@@ -0,0 +1,87 @@
+# Deploy RDS Aurora Cluster GitHub Action
+
+This GitHub Action automates the deployment of an Amazon RDS Aurora cluster using Terraform. It installs Terraform and AWS CLI, and outputs the Aurora cluster endpoint along with other relevant details.
+
+## Description
+
+The **Deploy RDS Aurora Cluster** action enables you to:
+
+- Automate the deployment of an RDS Aurora cluster on AWS.
+- Use Terraform for infrastructure as code.
+- Install specific versions of Terraform and AWS CLI.
+- Output the Aurora cluster endpoint, Terraform state URL, and all other Terraform outputs dynamically.
+
+## Inputs
+
+The following inputs are required or optional for the action:
+
+| Input | Description | Required | Default |
```

Comment:
general suggestion for the input list, if it has a default, is it really required?

Reply:
same for the actual `action.yml`, you define defaults and set them to required 🤷